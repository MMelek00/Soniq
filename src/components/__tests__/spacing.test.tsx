/**
 * @format
 */

import 'react-native';
import React from 'react';

import renderer from 'react-test-renderer';
import Spacing from '../spacing';
describe('GenericText', () => {
  it('renders correctly with Error', () => {
    const tree = renderer.create(
      <Spacing backgroundColor={'red'} height={16} />,
    );
    expect(tree).toMatchSnapshot();
  });
});
