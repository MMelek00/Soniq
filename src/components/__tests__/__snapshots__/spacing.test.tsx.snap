// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenericText renders correctly with Error 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "",
        "height": "",
        "width": "",
      },
      {
        "backgroundColor": "red",
        "height": 16,
        "width": undefined,
      },
    ]
  }
/>
`;
